tests/integration/test_start_available_mip_dna.py

```diff
@@ -0,0 +1,319 @@
+    yield store
+
+
+@pytest.fixture
+def housekeeper_db(
+    housekeeper_db_uri: str, tmp_path_factory: TempPathFactory
+) -> Generator[HousekeeperStore, None, None]:
+    hk_database.initialize_database(housekeeper_db_uri)
+    hk_database.create_all_tables()
+    housekeeper_root: Path = tmp_path_factory.mktemp("housekeeper")
+    store = HousekeeperStore(root=housekeeper_root.as_posix())
+    yield store
+    hk_database.drop_all_tables()
+
+
+@pytest.fixture
+def scout_export_panel_stdout() -> bytes:
+    return b"22\t26995242\t27014052\t2397\tCRYBB1\n22\t38452318\t38471708\t9394\tPICK1\n"
+
+
+@pytest.mark.integration
+def test_start_available_mip_dna(
+    helpers: StoreHelpers,
+    housekeeper_db_uri: str,
+    housekeeper_db: HousekeeperStore,
+    httpserver: HTTPServer,
+    mocker: Callable,
+    scout_export_panel_stdout: bytes,
+    status_db_uri: str,
+    status_db: Store,
+    tmp_path_factory: TempPathFactory,
+):
+    """Test a successful run of the command start-available mip-dna
+    with one case to be analysed that has not been analysed before"""
+    cli_runner = CliRunner()
+
+    # GIVEN a mip root directory
+    test_root_dir: Path = tmp_path_factory.mktemp("test_start_available_mip_dna")
+    # GIVEN a config file with valid database uris and directories
+    config_path: Path = create_config(status_db_uri, housekeeper_db_uri, test_root_dir)
+    mip_dna_path = Path(test_root_dir, "mip-dna")
+
+    # GIVEN a case with existing qc files
+    ticket_id = 12345
+    case: Case = helpers.add_case(
+        store=status_db, data_analysis=Workflow.MIP_DNA, ticket=str(ticket_id)
+    )
+    create_qc_file(test_root_dir, case)
+
+    # GIVEN an order associated with the case
+    order: Order = helpers.add_order(
+        store=status_db, ticket_id=ticket_id, customer_id=case.customer_id
+    )
+    status_db.link_case_to_order(order_id=order.id, case_id=case.id)
+
+    # GIVEN a sample associated with the case
+    sample: Sample = helpers.add_sample(
+        store=status_db, last_sequenced_at=datetime.now(), application_type=AnalysisType.WGS
+    )
+    helpers.relate_samples(base_store=status_db, case=case, samples=[sample])
+
+    # GIVEN a flow cell and sequencing run associated with the sample
+    flow_cell: IlluminaFlowCell = helpers.add_illumina_flow_cell(store=status_db)
+    sequencing_run: IlluminaSequencingRun = helpers.add_illumina_sequencing_run(
+        store=status_db, flow_cell=flow_cell
+    )
+    helpers.add_illumina_sample_sequencing_metrics_object(
+        store=status_db, sample_id=sample.internal_id, sequencing_run=sequencing_run, lane=1
+    )
+
+    # GIVEN a gzipped-fastq file exists
+    fastq_base_path: Path = tmp_path_factory.mktemp("fastq_files")
+    fastq_file_path: Path = Path(fastq_base_path, "file.fastq.gz")
+    shutil.copy2("tests/integration/config/file.fastq.gz", fastq_file_path)
+
+    # GIVEN bundle data with the fastq files exists in Housekeeper
```

Comment:
Not sure if it matters here, but fastq always come in pairs (R1 and R2). If you consider that it is important to consider this detail, then we shoudl include another one.